standard_forms/purchase-order/latest/src/header.html

```diff
@@ -0,0 +1,85 @@
+{% macro dateOrNA(datetime) %} {% if datetime %}{{ datetime |
+date(format="%d/%m/%Y", timezone=arguments.timezone) }}{% else
+%}{{t(k="messages.not-applicable", f="N/A")}}{% endif %} {% endmacro input %}
+
+<table class="header_image_section" style="width: 100%; height: 98%">
+  <tr>
+    <th style="text-align: left; width: 10%; height: 70%">
+      <img class="logo" src="data.store.logo" />
+    </th>
+    <th style="text-align: left; width: 20%">
+      <span style="font-weight: bold">
+        {{ data.purchaseOrder.store.name.name }}
+      </span>
+      <br />
+      <span style="font-size: 6pt">
+        {{ data.purchaseOrder.store.name.address1 }}
+      </span>
+      <br />
+      <span style="font-size: 6pt">
+        {{ data.purchaseOrder.store.name.address2 }}
+      </span>
+      <br />
+      <span style="font-size: 6pt">
+        {{t(k="label.phone",f="Telephone")}}: {{
+        data.purchaseOrder.store.name.phone }}
+      </span>
+      <br />
+      <span style="font-size: 6pt">
+        {{t(k="label.email",f="Email")}}: {{ data.purchaseOrder.store.name.email
+        }}
+      </span>
+    </th>
+    <th style="text-align: right; width: 70%; font-size: 28pt">
+      {{t(k="label.purchase-order", f="Purchase Order") }}<br />
+    </th>
+  </tr>
+</table>
+<div
+  class="header_supplied_section"
+  style="display: flex; flex-direction: column"
+>
+  <div class="header-space-between">
+    <span>
+      {% if data.purchaseOrder.supplier %} {{ t(k="label.supplier",
+      f="Supplier") }}: {% endif %}
+    </span>
+
+    <span> 
+        {{t(k="label.purchase-order-number", f="Purchase Order Number") }}: {{
+        data.purchaseOrder.number }}
+    </span>
+  </div>
+  <div class="header-space-between">
+    <span>
+      {% if data.purchaseOrder.supplier %} {{ data.purchaseOrder.supplier.name
+      }} {% endif %}
+    </span>
+
+    <span id="date">
+      {{ t(k="report.printed-date", f="Printed date") }}: {{ now() |
+      date(format="%d/%m/%Y") }}
+    </span>
+  </div>
+  <div class="header-space-between">
+    <span>
+      {% if data.purchaseOrder.supplier %} {{ data.purchaseOrder.supplier.code }}
+      {% endif %}
+    </span>
+    <span id="date"></span>
+      {{ t(k="report.confirmed-date", f="Confirmed date") }}: 
+      {{self::dateOrNA(datetime=data.purchaseOrder.confirmedDatetime)}}
+    </span>
+  </div>
+  <div class="header-space-between">
+    <span>
+      {% if data.purchaseOrder.supplier %} data.purchaseOrder.supplier.address1 {%
```

Comment:
Similarly here - data.purchaseOrder.supplier.address1 needs to be in {{}} brackets. 

Any time data is accessed, it needs to be in those brackets. 

Currently it is rendering a string:

![Screenshot 2025-06-30 at 3 21 00 PM](https://github.com/user-attachments/assets/499994cc-590e-4432-acc3-2b7e5d32fea3)